app/models/node.rb

```diff
@@ -110,4 +111,13 @@ def destroy_attachments
     attachments_dir = Attachment.pwd.join(self.id.to_s)
     FileUtils.rm_rf attachments_dir if File.exists?(attachments_dir)
   end
+
+  def parent_node
+    if self.parent.nil? ||
+      !(self.parent.type_id == Node::Types::DEFAULT ||
```

Comment:
This approach means we need to tweak this line if in the future we add a new Type.

The same list of types is being used by the `#user_nodes` scope. Should we extract into a helper method and reuse in both places?